demo/.eslintrc.js

```diff
@@ -14,8 +14,9 @@ module.exports = {
     "prettier/react",
   ],
   rules: {
-    "no-console": 0,
-    "react/prop-types": 0,
+    "no-console": "off",
```

Comment:
I think in the long run we want `no-console` on or at least allowing `console.warn` and `console.error`. If we have it on, we can make a function that wraps the console.log functions and only uses them when a dev env var is set to true or something like that. But either way, this change to strings for the config rather than integers is a great change!


https://eslint.org/docs/rules/no-console

Reply:
This can wait until a future issue, though https://github.com/tasitlabs/tasit/issues/28